wagtail/core/templatetags/wagtailcore_tags.py

```diff
@@ -19,6 +20,9 @@ def pageurl(context, page):
     if not hasattr(page, 'relative_url'):
         raise ValueError("pageurl tag expected a Page object, got %r" % page)
 
+    if getattr(context['request'], 'is_preview', False):
+        return reverse('wagtailadmin_pages:view_draft', args=[page.id])
+
```

Comment:
Worth noting here that 'preview' and 'draft' are two distinct concepts - the preview button on the edit page shows the result of the edits the user has just made, which have not been saved to the database at that point (not even as draft). As a result, this won't have the intended effect - `is_preview` will become false when you follow the first link, and so the links on subsequent pages will revert to the ordinary live URLs.

Reply:
Ah. Is there an equivalent `is_draft`?